### PR TITLE
Updates the GoogleTest and Google Benchmark dependencies

### PR DESCRIPTION
**Pull request by derekmauro**
This change is [Reviewable](https://reviewable.io/reviews/google/cctz/192)

### CODE REVIEW

**Comment by derekmauro** [comment]
I'm taking a look at the CI failures.

File: WORKSPACE [hunk 1]
```diff
@@ -5,26 +5,23 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
...
     name = "rules_cc",
-    sha256 = "fa42eade3cad9190c2a6286a6213f07f1a83d26d9f082d56f526d014c6ea7444",
-    strip_prefix = "rules_cc-02becfef8bc97bda4f9bb64e153f1b0671aec4ba",
-    urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_cc/archive/02becfef8bc97bda4f9bb64e153f1b0671aec4ba.zip",
-        "https://github.com/bazelbuild/rules_cc/archive/02becfef8bc97bda4f9bb64e153f1b0671aec4ba.zip",
-    ],
+    sha256 = "f3908cb40a6577ab0d1ef9e00052739bf69b4313fa7d20b4d61d4521ea67abf3",
+    strip_prefix = "rules_cc-ab5395627c80e025e824bd005d41f96b20618b9d",
+    urls = ["https://github.com/bazelbuild/rules_cc/archive/ab5395627c80e025e824bd005d41f96b20618b9d.zip"],
```
Comment (devbww) [inline WORKSPACE:26]: I used the current head (daf6ace7cfeacd6a83e9ff2ed659f416537b6c74) for this one.  Any reason not to?

Comment (derekmauro) [reply WORKSPACE]: I actually just reverted this. It looks like we need to update the Bazel version of Windows/Mac before we can update this.

Comment (devbww) [reply WORKSPACE]: Ack.

**Review by devbww — APPROVED** [review]

**Pull request merged**